script/zonemaster_backend_testagent

```diff
@@ -60,7 +61,7 @@ $opt_outfile //= '/var/log/zonemaster/zonemaster_backend_testagent.out';
 $loglevel    //= 'info';
 $loglevel = lc $loglevel;
 
-$loglevel =~ /^(?:trace|debug|info|inform|notice|warning|warn|error|err|critical|crit|fatal|alert|emergency)$/ or die "Error: Unrecognized --loglevel $loglevel\n";
+die "Error: Unrecognized --loglevel $loglevel\n" unless exists $CanonicalLevelNames{$loglevel};
```

Comment:
Just to point out that this will remove the following levels: `fatal`, `inform` and `trace`.

Reply:
but using one of those levels was making the testagent crash later anyway.

Reply:
Yes this is the reason for changing this check.